codegen/tests/simple/skeletonhandlerimpl.go

```diff
@@ -0,0 +1,58 @@
+// Code generated by sysl DO NOT EDIT.
```

Comment:
nitpick: I think perhaps naming the file `skeletonhandler.go` would be cleaner and more go-like?

Reply:
removed.